templates/zerver/features.html

```diff
@@ -301,7 +301,7 @@
                     <div class="block">
                         <h4>{% trans %}Desktop apps{% endtrans %}</h4>
                         <p>{% trans %}Prefer Zulip in its own window and rich, OS-level notifications?
-                          Enjoy <a href="/apps">Zulip on your desktop</a>.{% endtrans %}</p>
+                          Enjoy <a href="https://zulipchat.com/apps">Zulip on your desktop</a>.{% endtrans %}</p>
```

Comment:
I think we might want these to still be relative links if ZILENCER_ENABLED; since e.g. on zulipchat.com (or anything else with ZILENCER_ENABLED and REALMS_HAVE_SUBDOMAINS), we probably want to keep you yourSubdomain.zulipchat.com, not move you to the root domain. 

So probably the right mechanism is to define a variable in `zproject/context_processors` for the apps_page, and have that variable be a relative link (if ZILENCER_ENABLED) or this absolute link otherwise.